### ISSUE TITLE
validation set and test set issue

### ISSUE THREAD

**Issue by huangchaoxing** [LossNAN/I3D-Tensorflow#27]
When I was reading your code, it seems that you put the test data into the validation process during training, which might not be right. shouldn't we split some data from the training list as the validation set ?

**Comment by huangchaoxing** [comment]
Also, it seems that the data list used in rgb and optical flow training are all "flow_list"....we shall use rgb_list in rgb case.
